cms/djangoapps/contentstore/views/course.py

```diff
@@ -575,6 +575,10 @@ def filter_ccx(course_access):
 
     if course_keys:
         courses_list = CourseOverview.get_all_courses(filter_={'id__in': course_keys})
+    else:
+        # If no course keys are found for the current user, then return without filtering
+        # or ordering the courses list.
+        return courses_list, []
```

Comment:
This is the behavior this function had before https://github.com/openedx/edx-platform/pull/34173, you can review it here:
https://github.com/openedx/edx-platform/pull/34173/files#diff-937adee5129f53ead6b3c7c988d70db9e9e95862f17a6f85a1eea2388d12b461R575-L512